.github/workflows/maven.yml

```diff
@@ -1,6 +1,6 @@
 name: Github CI Build
 
-on: [push]
+on: [push, workflow_dispatch]
```

Comment:
Go to your project's GitHub homepage, click "Actions". You will see a "Run workflow" button. The thing is, that the button only becomes visible after the corresponding workflow containing the `on: workflow_dispatch` option has been committed to master. After you have merged my PR now, you should be able to see it. Here is a screenshot from my own fork, because I have no rights to start workflows in your repository (unless you want to give rights to me):

![image](https://user-images.githubusercontent.com/1537384/149645414-23e89d13-77f9-4b51-8c36-6ddde2566b23.png)

Reply:
BTW, I just clicked your profile for the first time. I see we are both German and even both live in the Nürnberg area (if I am in Germany, because I travel a lot). If you like, you can contact me on [Gitter](https://gitter.im/), then we can also have non-public conversations.